overview.md

```diff
@@ -44,23 +50,26 @@ Scope Frameworks define the rules that SubDAOs must adhere to. Scope Frameworks
 
-The objective of DVCs is to ensure that they represent MKR voters that are aligned with the values of that DVC. Members of DVCs will be compensated for their engagement under certain conditions, thereby incentivizing active governance. 
+The objective of CVCs is to ensure that they represent MKR voters that are aligned with the values of that CVC. Members of CVCs will be compensated for their engagement under certain conditions, thereby incentivizing active governance. 
+
+CVCs work together with [Delegates](maker-core/delegates.md) as well as [Advisory Councils](maker-core/advisory-councils.md) to implement specific governance changes.
 
 ### [Tokenomics](tokenomics/launch-tokenomics.md)
 
-The Endgame Plan introduces [Elixir](tokenomics/elixir.md) and [SubElixir](tokenomics/subelixir.md), which are liquidity pool tokens representing liquidity pools holding various combinations of MKR, ETH, DAI and SubDAO governance tokens. The aim of these tokens is twofold. First, it ensures alignment of interests between Maker Core and SubDAOs. Second, it also improves liquidity for all these tokens on the open market. 
+The Endgame Plan introduces [Elixir](tokenomics/elixir.md) and [SubElixir](tokenomics/subelixir.md), which are liquidity pool tokens representing liquidity pools holding various combinations of MKR, Dai and SubDAO governance tokens. The aim of these tokens is twofold. First, it ensures alignment of interests between Maker Core and SubDAOs. Second, it also improves liquidity for all these tokens on the open market. 
 
 The Endgame Plan has two major tokenomics updates. First, annual MKR minting will be used to incubate SubDAOs, fund DVCs, and incentivize the workforce. These emissions counteract the MKR burn and reduce the likelihood of MKR becoming concentrated in the hands of large holders.
 
-ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDAI](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDAI is one of the central products of the Endgame Plan. 
+ETH and staked ETH will play a major role in ensuring the financial resilience of Maker Core. [EtherDai](tokenomics/etherdai.md) is a synthetic Ether token backed by liquid staked Ether tokens and Maker-issued debt. EtherDai is one of the central products of the Endgame Plan. 
+
+In the final stages of Endgame, Dai will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
 
-In the final stages of Endgame, DAI will turn into an Unbiased World Currency. In particular, it is not expected to remain pegged to the US Dollar and will instead have alternative mechanisms that determine its Target Price. 
+MKR tokens will no longer function as a backstop in the event that Maker is insolvent - instead, this will be reflected in the Target Price of Dai. 
```

Comment:
Not strictly accurate. It will by default, unless governance decide it shouldn't.